easy_my_coop_export_xlsx/__manifest__.py

```diff
@@ -27,8 +27,8 @@
     """,
     'author': 'Houssine BAKKALI, <houssine@coopiteasy.be>',
     'license': 'AGPL-3',
-    'version': '9.0.1.0',
-    'website': "www.coopiteasy.be",
+    'version': '9.0.1.0.0',
```

Comment:
@remytms should this version start with `12`?

Reply:
@enricostano You're right. :) It's done.